src/url.js

```diff
@@ -57,6 +57,10 @@ export function parseUrl(url) {
     return fromCache;
   }
   a.href = url;
+  // IE11 doesn't provide full URL components when parsing relative URLs.
+  // Assigning to itself again does the trick.
+  a.href = a.href;
```

Comment:
What are consequences for other browsers? Can we only do it in some condition? E.g. when we can confirm that the URL is not absolute? Or did you check that it's a no-op on other browsers performance-wise?